src/dotnet/Fable.Compiler/FSharp2Fable.fs

```diff
@@ -1122,7 +1238,7 @@ type FableCompiler(com: ICompiler, state: ICompilerState, checkedProject: FSharp
     member fcom.UsedVarNames = set usedVarNames
     interface IFableCompiler with
         member fcom.Transform ctx fsExpr =
-            transformExpr fcom ctx fsExpr
+            transformExpr [] fcom ctx [fsExpr]|> List.head
```

Comment:
I'm a bit concerned about this, as we're bypassing the type checker (we're assuming that the list will always be one item) and it could cause silent bugs in the future. Several places use the `Transform` active pattern which calls this method.